common/src/key_range.rs

```diff
@@ -1,10 +1,18 @@
 use bytes::Bytes;
 
+#[derive(Clone, Debug)]
 pub struct KeyRange {
     pub lower_bound_inclusive: Option<Bytes>,
     pub upper_bound_exclusive: Option<Bytes>,
 }
 
+impl PartialEq for KeyRange {
```

Comment:
Is there a reason why you are implementing this yourself instead of deriving PartialEq?

Reply:
Duh. I didn't realize that it was an option, smh.